packages/blade/src/components/Input/OTPInput/OTPInput.stories.tsx

```diff
@@ -1,9 +1,9 @@
 import type { ComponentStory, Meta } from '@storybook/react';
 import { Title, Subtitle, Primary, ArgsTable, Stories, PRIMARY_STORY } from '@storybook/addon-docs';
-import { Highlight } from '@storybook/design-system';
 import React from 'react';
 import type { OTPInputProps } from './OTPInput';
 import { OTPInput as OTPInputComponent } from './OTPInput';
+import Sandbox from '~src/_helpers/storybook/Sandbox';
```

Comment:
Looks like Figma embeds are missing for all Inputs & Spinners. Can we add that too?

Reply:
So I am adding this refactor as well #822 to standardize things like these. Will add the missing links there since it'll be easier to pass links there.